wiki/Glossary/Local_song_offset/fr.md

```diff
@@ -15,6 +15,10 @@ Au début du jeu, le local song offset peut être modifié en appuyant sur :
 - `Alt` + `+` pour augmenter le décalage de 1 milliseconde
 - `Alt` + `-` pour diminuer le décalage de 1 milliseconde
 
-Si un local offset est défini, osu! affichera le local offset dans l'interface au-dessus du tableau d'affichage. osu! vous informera également du local offset avant de commencer le jeu.
+Si un local offset est défini, osu! affichera le local offset dans l'interface au-dessus du tableau d'affichage.
 
-<!-- TODO: Add links and stuff -->
+![Local Offset](img/local-offset.jpg "Décalage local")
```

Comment:
```suggestion
![Décalage local](img/local-offset.jpg "Décalage local")
```

Reply:
@Pisapou @Biokax the article title doesn't seem to be translated, is that an oversight?

Reply:
I don't think so, "offset" just sounds better untranslated, but the fact that it has been translated here is a mistake (between quotation mark)

Reply:
```suggestion
![Local offset](img/local-offset.jpg "Local offset")
```